example/reinforcement-learning/a3c/rl_data.py

```diff
@@ -21,13 +21,18 @@
 import gym
 import cv2
 import math
-import Queue
 from threading import Thread
 import time
 import multiprocessing
 import multiprocessing.pool
 from flask import Flask, render_template, Response
 import signal
+import sys
+is_py3 = sys.version[0] == '3'
+if is_py3:
+    import queue as queue
+else:
+    import Queue as queue
```

Comment:
I could not reproduce the error while running the example, but I have added the version based condition.